benchmark/projection/benchmark.cpp

```diff
@@ -0,0 +1,90 @@
+#include <benchmark/benchmark.h>
+#include <sycl/sycl.hpp>
+
+namespace Nektar::LibUtilities {
```

Comment:
Can we do
```
#include <LibUtilities/BasicUtils/ShapeType.hpp>
```
and avoid the duplicate definition?

Reply:
The benchmarks or (the projection code) don't depend on nektar except for those definitions, or anything else really except sycl and the benchmark suite - in theory though not in practice unless I fiddle with the cmake a bit. Not really that important but I like to keep things seperete if I can - looking at it now I could probably just take it all out. It only comes up here 
 
 https://github.com/ExCALIBUR-NEPTUNE/NESO/blob/445305222374aef420dde765c3184d5041e10cbf/include/nektar_interface/function_basis_projection_alt.hpp#L60

and I could translate in that file instead.

Reply:
I still think this is just asking for duplicate declaration errors if any other header file uses Nektar in future.